### PR TITLE
updated workshop docs

### LINKED ISSUE #545: Creating SQL tables (Optional): Sending the data via Trino notebook, accessing and interacting with the dataset through Cloudbeaver UI.

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies

closes #545 
#540 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Updated JB, added more content, screenshot and brought workshop section to the top

### CODE REVIEW

**Comment by oindrillac** [comment]
@aakankshaduggal @suppathak updated docs. PTAL :)

File: docs/workshop/model_development.md [hunk 1]
```diff
@@ -43,16 +51,22 @@ For faster download, choose a repository which does not have too many PRs. For e
...
+    REMOTE=1
 
 ```
 
 ## Model Development
 
 In this workshop, we will be training a machine learning model using tools and services available on Operate First.
 
-The aim of this model is to take a Github repository of interest and predict the time that it will take to merge a new Pull Request. For that purpose, we frame the “time taken to merge a PR” as a classification problem where we predict whether the time taken to merge a PR falls within one of a few predefined time ranges.
+To directly see the model development running in a pipeline or in action, skip to the [next section](./ml-pipeline.md)
```
Comment (suppathak) [inline docs/workshop/model_development.md:62]: Link to the `next section` not working.

**Review by suppathak — COMMENTED** [review]
Looks good! Just added a minor comment on typo.

File: docs/workshop/model_development.md (hunk 1 above)
Comment (oindrillac) [reply docs/workshop/model_development.md]: updated, thanks :)

File: docs/workshop/model_development.md [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
 * Open the terminal and clone your forked Github repository in Jupyterhub.
 
 `git clone https://github.com/{YOUR-USERNAME}/ocp-ci-analysis`
 
-* Set up env variables at the root of the repository.
+If you do not have a Github account or have not forked the repository, move to chapter on [Set up Git environment](./git_setup.md).
+
+* Set up environment variables at the root of the repository.
+
+Once you have cloned the repository, [cd](https://linuxize.com/post/linux-cd-command/) into the repository and create a `.env` file.
```
Comment (suppathak) [inline docs/workshop/model_development.md:23]: @oindrillac . I missed to mention previously. But I think we need to add a instruction to configure their git repo. 

```
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
```

Comment (oindrillac) [reply docs/workshop/model_development.md:23]: good point will add this

File: docs/workshop/model_development.md [hunk 3]
```diff
@@ -1,29 +1,37 @@
...
+In this section, we will learn how to train an AIOps model from scratch. The model that we plan to train in particular is a Time to Merge prediction model which offers time estimates on when a Pull Request on a Git repository will be merged. To train such a model we look at historic pull request data of the repository (can also be extended to organization) and engineer features from the Pull Requests which can impact the time to merge of a Pull Request.
+
+To train such a model, we will be using a Jupyterlab environment and make use of jupyter notebooks. The following steps will walk you through how to train a Github time to merge model.
 
 ## Set Environment Variables
 
-This is an important step to get right for successfully being able to run the notebooks.
+This is an important step to get right for successfully being able to run the jupyter notebooks.
+
+* Go to [Jupyterhub](https://jupyterhub-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/hub/spawn) and select the ocp ci analysis image and the “Workshop” container size.
```
Comment (aakankshaduggal) [inline docs/workshop/model_development.md:11]:
```suggestion
* Go to [Jupyterhub](https://jupyterhub-aiops-tools-workshop.apps.smaug.na.operate-first.cloud/hub/spawn) and select the `Openshift CI Analysis Notebook Image` and the “Workshop” container size.
```

Comment (oindrillac) [reply docs/workshop/model_development.md]: changed! thanks :)

File: docs/workshop/sql_query_engine.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
-# SQL Query Engine - Trino
+# Create SQL tables from pull requests dataset
 
-This section will include instructions for sending the data via Trino notebook, accessing and interacting with the dataset through Cloudbeaver UI.
+We can also create interactive visualizations and charts on [Apache Superset](https://superset.apache.org/) using the data that we collect from Github Repositories.
```
Comment (aakankshaduggal) [inline docs/workshop/sql_query_engine.md:3]: Should we add the opf workshop url for Superset here to avoid confusion for folks?

**Review by aakankshaduggal — COMMENTED** [review]
Looks good, just left few nits :)

**Comment by oindrillac** [comment]
@aakankshaduggal @suppathak made requested changes. PTAL

**Pull request merged**